boards.txt

```diff
@@ -1991,3 +1991,25 @@ Midatronics.menu.rtlib.nanofps=Newlib Nano + Float Printf/Scanf
 Midatronics.menu.rtlib.nanofps.build.flags.ldspecs=--specs=nano.specs -u _printf_float -u _scanf_float
 Midatronics.menu.rtlib.full=Newlib Standard
 Midatronics.menu.rtlib.full.build.flags.ldspecs=
+
+###############################
+# ThunderPack
+ThunderPack.name=ThunderPack
```

Comment:
Please create a new `Generic L0 series` menu with ThunderPack as a submenu like this further L0 based board addition could be add here.


```suggestion
GenL0.name=Generic STM32L0 series
...
```
then 
```
GenL0.menu.pnum.THUNDER_PACK=ThunderPack
...
```

Reply:
✔️